docs/tutorials/demo.ipynb

```diff
@@ -469,11 +432,11 @@
     "# If we get an error with direct_access=True, most likely is because\n",
```

Comment:
`direct_access=True` isn't in this code block though, so where is that coming from?

Reply:
I'm refactoring the notebook to use the top level API and this should go away!